barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_flavor.hpp

```diff
@@ -278,6 +278,7 @@ class UltraFlavor {
           // shifted polys
             ZoneScopedN("creating empty prover polys");
             for (auto& poly : get_unshifted()) {
+                // WORKTODO(sparse): proper lower bound on size
```

Comment:
Flagging this WORKTODO. Fine with me if you want to keep them here for now assuming this is going to be a continued thread of work

Reply:
Removed this, we'll go over all constructors